consensus/core/src/network/tonic_network.rs

```diff
@@ -587,6 +644,24 @@ pub(crate) struct FetchBlocksResponse {
     blocks: Vec<Bytes>,
 }
 
+#[derive(Clone, prost::Message)]
+pub(crate) struct FetchCommitsRequest {
+    #[prost(uint32, tag = "1")]
+    start: CommitIndex,
+    #[prost(uint32, tag = "2")]
+    end: CommitIndex,
+}
+
+#[derive(Clone, prost::Message)]
+pub(crate) struct FetchCommitsResponse {
+    // Serialized Commit.
+    #[prost(bytes = "bytes", repeated, tag = "1")]
+    commits: Vec<Bytes>,
+    // Serialized SignedBlock.
+    #[prost(bytes = "bytes", repeated, tag = "2")]
+    certifier_blocks: Vec<Bytes>,
```

Comment:
nit: same as anemo, `signed_blocks`?

Reply:
Similarly I want to indicate that the blocks are for certifying the commits.